definition-xml/ardupilotmega.xml

```diff
@@ -2001,4 +2009,4 @@
       <field type="uint16_t[4]" name="count">count of telemetry packets received (wraps at 65535).</field>
     </message>
   </messages>
-</mavlink>
+</mavlink>
```

Comment:
nitpick: generally recommended to put new lines and the end of files like this, probably won't matter here